www/data_module/src/mock/generator.service.coffee

```diff
@@ -0,0 +1,31 @@
+# types for generating test data: null, number, string, boolean, timestamp, <array>[], <object>, <objectName in Specification>
+class Generator extends Service
+    self = null
+    constructor: ->
+        self = @
+
+    number: (min = 0, max = 100) ->
+        random = Math.random() * (max - min) + min
```

Comment:
I'd like to use this generator to help the offline development. In unit tests I find myself to write exact  test data for that scenario where a generator won't be really useful even if it were reproducible.